src/browser/browser.ts

```diff
@@ -137,7 +137,7 @@ export class Browser {
       launcherOptions.executablePath = this.config.chromeBin;
     }
 
-    launcherOptions.headless = !this.config.headed;
+    launcherOptions.headless = !this.config.headed ? "shell" : false;
```

Comment:
shell is for the old headed [mode](https://developer.chrome.com/blog/chrome-headless-shell)